### PR TITLE
7 sooiuoc implementation

### PR DESCRIPTION
**Pull request by act-reds**

### CODE REVIEW

**Review by daniel-rossier — APPROVED** [review]

File: agency/usr/mqtt_iuoc/mqtt_iuoc_client.cpp [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+std::vector<std::string> topics_publish_name(IUOC_ME_END);
+
+std::vector<std::string> pub_debug;
+
+std::string get_payload_from_data(iuoc_data_t *data);
+
+int main(int argc, char* argv[])
+{
+	//test_ioctl();
+	std::ifstream inFile;
```
Comment (jmi-reds) [inline agency/usr/mqtt_iuoc/mqtt_iuoc_client.cpp:53]: alignment

File: agency/usr/mqtt_iuoc/mqtt_iuoc_client.cpp [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+	}
+
+	std::string server_address = root["server_address"].asString();
+	std::string client_id = root["client_id"].asString();
+
+	// Mapping for topics and MEs
+	std::unordered_map<std::string, int> sub_topics_mapping;
+	std::unordered_map<int, std::string> pub_topics_mapping;
+
+	// sub_topics_mapping.insert()
```
Comment (jmi-reds) [inline agency/usr/mqtt_iuoc/mqtt_iuoc_client.cpp:85]: To remove or comment

**Review by jmi-reds — COMMENTED** [review]
I checked IUOC src files only. 
there are many places where there is a mix between tab and space for the alignment !

**Comment by jmi-reds** [comment]
On the user mqttt app, the source files have a mix between space and tab !

**Pull request merged**